packages/appium/lib/main.js

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env node
 
 // transpile:main
```

Comment:
btw, with the removal of gulp, i think we can get rid of all of these `//transpile` comments?